src/admin.rs

```diff
@@ -67,6 +72,11 @@ where
             trace!("RESUME");
             resume(stream, query_parts[1]).await
         }
+        "SHUTDOWN" => {
+            trace!("SHUTDOWN");
+            is_shutdown = true;
```

Comment:
Why don't you just send `TERM` signal to itself here?

```
let pid = std::process::id;
signal::kill([pid, Signal::TERM);
```

Reply:
That's a good idea, I originally thought doing this would pick up the PID of the worker thread but it seems to get the main thread PID.

I've updated the implementation to use that instead, thanks!